airflow/models/crypto.py

```diff
@@ -17,8 +17,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from typing_extensions import Protocol
-from typing import Optional
+from airflow.typing import Protocol
+from typing import Optional  # noqa
```

Comment:
This shouldn't be needed as Optional is used in the type comments which counts as used for all the tools we use in CI.

```suggestion
from typing import Optional
```